indexer/index.go

```diff
@@ -36,24 +35,14 @@ var (
 	errNumToFetchInvalid   = fmt.Errorf("numToFetch must be in [1,%d]", MaxFetchedByRange)
 	errNoContainerAtIndex  = errors.New("no container at index")
 
-	_ Index = (*index)(nil)
+	_ snow.Acceptor = (*index)(nil)
 )
 
-// Index indexes containers in their order of acceptance
-// Index is thread-safe.
-// Index assumes that Accept is called before the container is committed to the
-// database of the VM that the container exists in.
-type Index interface {
-	snow.Acceptor
-	GetContainerByIndex(index uint64) (Container, error)
-	GetContainerRange(startIndex uint64, numToFetch uint64) ([]Container, error)
-	GetLastAccepted() (Container, error)
-	GetIndex(id ids.ID) (uint64, error)
-	GetContainerByID(id ids.ID) (Container, error)
-	io.Closer
```

Comment:
nit: should we add an interface check for io.Closer?

Reply:
We never use `index` as a `Closer`